### DISCUSSION TITLE
[Kavita] Add link to UI translations page info in wiki

### DISCUSSION THREAD

**Discussion by shimizurei** [Kareadita/Kavita#2602] [Ideas]
### Idea Description

It'd be nice getting people interested in translating the UI with weblate.

Feel free to leave comments with your opinions, suggestions, or even constructive critiques.

### Duration of Using Kavita

?? For a few years, I think?

### Idea Category

User Experience

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
Forgot that I implemented this in v0.7.14. Thanks for the suggestion. :)
